http/1.5/modules/ROOT/nav.adoc

```diff
@@ -1,24 +1,25 @@
 .xref:index.adoc[HTTP Connector]
 * xref:index.adoc[About HTTP Connector]
-* xref:http-about-http-request.adoc[HTTP Requests and Responses]
-* xref:http-authentication.adoc[Authenticate HTTP Requests]
+** xref:http-listener-ref.adoc[HTTP Listener Configuration Reference]
+** xref:http-request-ref.adoc[HTTP Request Configuration Reference]
+** xref:http-about-http-request.adoc[HTTP Requests and Responses]
+** xref:http-authentication.adoc[Authenticate HTTP Requests]
+** xref:http-about-loading-static-resource.adoc[About Loading a Static Resource]
+** xref:http-about-http-connector-migration.adoc[HTTP Connector Migration]
 * xref:http-about-http-connector.adoc[HTTP Connector (Design Center)]
-* xref:http-trigger-app-from-browser.adoc[Start an App from a Browser]
-* xref:http-consume-web-service.adoc[Consume a REST Service]
-* xref:http-create-https-listener.adoc[Create an HTTPS Listener]
-* xref:http-basic-auth-task.adoc[Set Up Basic Authentication]
 * xref:http-connector-studio.adoc[HTTP Connector (Studio)]
-* xref:http-start-app-brows-task.adoc[Start an App from a Browser]
-* xref:http-consume-web-svc-task.adoc[Consume a REST Service]
-* xref:http-load-static-res-task.adoc[Load a Static Resource]
-* xref:http-conn-route-diff-paths-task.adoc[Route HTTP Requests to Different Paths]
-* xref:http-route-methods-based-task.adoc[Route Requests Based on HTTP Method]
-* xref:http-post-form-task.adoc[Post Form]
-* xref:http-about-loading-static-resource.adoc[About Loading a Static Resource]
-* xref:http-listener-ref.adoc[HTTP Listener Configuration Reference]
-* xref:http-request-ref.adoc[HTTP Request Configuration Reference]
+* xref:http-connector-examples.adoc[HTTP Connector Examples]
```

Comment:
Are any of these examples Design Center? If so, make a different section, so it would be like: 
HTTP Connector Examples
   Studio Examples
   Design Center Examples

Reply:
Yes, they are in the section called 
== HTTP Connector Design Center Examples